### ISSUE TITLE
App: Firefox 'not supported' should be updated.

### ISSUE THREAD

**Issue by ohmed** [webaverse/app#2283]
Current message about not supporting of FF and IE is misleading a bit,
telling 'web workers' are not supported [what is not true].

So suggesting to make it more informative, and maybe needs designs ?
Error message screen attached.

![image](https://user-images.githubusercontent.com/1478866/151240636-de113b77-182b-42ed-b066-764b2e704a0c.png)

**Comment by avaer** [comment]
I agree, we should get a general UI design for all of these dialogs and unify it.

**Comment by avaer** [comment]
> Current message about not supporting of FF and IE is misleading a bit,
telling 'web workers' are not supported [what is not true].

It says that `WebWorker modules` are not supported. It is a feature of Web Workers and I think the statement is accurate, but it is clearly misleading.

https://caniuse.com/mdn-api_worker_worker_ecmascript_modules

**Comment by 0reo** [comment]
@ohmed is this actually a UI task?
